lib/README.md

```diff
@@ -2,24 +2,25 @@
 
 Node.js module that serves Coveo search tokens for Dynamics portal.
 
-``` javascript
+```javascript
 // Decodes the authentication token from portal to extract the payload related to the authenticated user.
 const portalAuth: IDecodedPortalAuthTokenPayload = await portal.decodeAuthToken(req.headers.authorization);
 
 // Gets a search token from Coveo for the user specified in the token.
```

Comment:
Since the method name is `fetchSearchToken`, I would start the comment with `Fetches ...`.